src/playground/auth/auth.module.ts

```diff
@@ -74,7 +74,7 @@ import { NbAuthGuard } from './auth-guard.service';
 
         NbPasswordAuthStrategy.setup({
           name: 'email',
-
+          requireValidToken: false,
```

Comment:
I would say we need to introduce this not on a strategy level, but on an endpoint level.
For instance, in a case when login endpoint requires to have a valid token, but register endpoint doesn't.